src/driller.py

```diff
@@ -1,9 +1,9 @@
-""" Uses Pydriller to extract information from Github."""
+"""Uses Pydriller to extract information from Github."""
 from pydriller import RepositoryMining
 
```

Comment:
It seems complex to have this in its own file with such a small amount of code.

Reply:
This already exists in the master branch. We only made style guide fixes to pass the build.